test/lib/test-helper.ts

```diff
@@ -6,59 +6,101 @@ import Promise = require('bluebird');
 import io = require('socket.io-client');
 
 var log = debug('child_server:stdout');
+// CONST
+var SESSION_EXPIRATION: number = 3;
+var MAXBUFFERSIZE: number = 3;
+var POLL_TIMEOUT: number = 1000;
+
+function createCertificate(): Promise<any>
+{
+    return (Promise.promisify(child.execFile))('./scripts/gen_keys.sh', ['./temp']);
+}
+
+function removeCertificate(): Promise<any>
```

Comment:
Since this is test code, if there is an API, I would just make this synchronous.